chia/cmds/data_funcs.py

```diff
@@ -109,6 +131,21 @@ async def submit_pending_root_cmd(
     return res
 
 
+async def submit_all_pending_roots_cmd(
+    rpc_port: Optional[int],
+    fee: Optional[str],
+    fingerprint: Optional[int],
+    root_path: Optional[Path] = None,
+) -> Dict[str, Any]:
+    final_fee = None if fee is None else uint64(int(Decimal(fee) * units["chia"]))
```

Comment:
That intermediate `int()` call was an artifact from mis-hinting of the fixed width integer types which has been fixed.

```suggestion
    final_fee = None if fee is None else uint64(Decimal(fee) * units["chia"])
```